cmd/update.go

```diff
@@ -277,6 +278,9 @@ func getUserAgent(mode string) string {
 		}
 	}
 
+	uaAppend(" CPU ", fmt.Sprintf("(total_cores:%d; vendor:%s; family:%d; model:%d; stepping:%d; model_name:%s)",
+		runtime.NumCPU()*cpuid.CPU.PhysicalCores, cpuid.CPU.VendorString, cpuid.CPU.Family, cpuid.CPU.Model, cpuid.CPU.Stepping, cpuid.CPU.BrandName))
```

Comment:
`runtime.NumCPU()*cpuid.CPU.PhysicalCores` for sure isn't correct. `runtime.NumCPU()` returns the number of "logical CPUs", ie execution threads - presumably across all physical cpus in total.

Example:
```
runtime.NumCPU(): 32

cpuid.CPU.BrandName: AMD Ryzen 9 3950X 16-Core Processor
cpuid.CPU.PhysicalCores: 16
cpuid.CPU.ThreadsPerCore: 2
cpuid.CPU.LogicalCores: 32
```